pkg/config/config.go

```diff
@@ -225,6 +261,12 @@ func (s *Site) GetAbsPath(home string) (string, error) {
 	return cleanPath(home, s.Path)
 }
 
+// GetAbsContainerPath gets the directory for a site’s
+// container path.
+func (s *Site) GetAbsContainerPath(home string) (string, error) {
+	return cleanPath(home, s.Path+"/"+s.GetContainerPath())
```

Comment:
Instead of "/" we should use `os.PathSeparator` const to ensure cross platform compatibility (documentation https://pkg.go.dev/os#pkg-constants)